titus-server-gateway/src/main/java/com/netflix/titus/gateway/service/v3/internal/ExtendedJobSanitizer.java

```diff
@@ -93,6 +99,24 @@ private boolean isInNonCompliantWhiteList(com.netflix.titus.api.jobmanager.model
         return markNonCompliant(sanitizedJobDescriptor, nonCompliant);
     }
 
+    /**
+     * Jobs with entry point binaries containing spaces are likely relying on the legacy shell parsing being done by
```

Comment:
Would this break on `/bin/bash -c "cat | grep grep 'Wed May 16 20:47:08 PDT 2018'"`

Reply:
Or not break, but consider it "invalid?

Reply:
It depends how that is mapped onto `List<String> entryPoint` and `List<String> command`.

Valid:
```
entryPoint = ["/bin/bash", "-c"]
command = ["cat | grep grep 'Wed May 16 20:47:08 PDT 2018'"]
```

noncompliant:
```
entryPoint = ["/bin/bash -c "cat | grep grep 'Wed May 16 20:47:08 PDT 2018'"]
```

Reply:
This may be a corner case, but `["/apps/foo\ bar"]`

Reply:
I mention that in the comment, that spaces is _likely_ a problem, but not necessarily. The point of marking these jobs as `titus.noncompliant` is that we can reach out and help people fix the bad cases before we remove the old code path